public/configs/otherMarketParameters.json

```diff
@@ -480,7 +480,7 @@
     "referencePrice": 0.20404992804891367,
     "numOracles": 6,
     "liquidityTier": 2,
-    "assetName": "STEPN",
+    "assetName": "GMT",
```

Comment:
@yujin512 is this the actual assetName

Reply:
[Coinmarketcap](https://coinmarketcap.com/currencies/green-metaverse-token/) shows GMT (Green Metaverse Token) and STEPN seems to be the name of the app which is why Kaiko shows STEPN. It's a bit confusing but I think it can be either way.